spec.html

```diff
@@ -1211,15 +1271,29 @@ <h1>ToElementDescriptor ( _elementDescriptor_ )</h1>
           1. If _placement_ is `"prototype"` or `"static"`, throw a *TypeError* exception.
         1. If _kind_ is `"field"`,
           1. If _descriptor_ has a [[Get]], [[Set]] or [[Value]] internal slot, throw a *TypeError* exception.
-        1. Let _element_ be {[[Kind]]: _kind_, [[Key]]: _key_, [[Placement]]: _placement_, [[Descriptor]]: _descriptor_}.
-        1. If [[Kind]] is `"field"`, set _element_.[[Initializer]] to _initializer_.
+        1. Let _element_ be the ElementDescriptor { [[Kind]]: _kind_, [[Key]]: _key_, [[Placement]]: _placement_, [[Descriptor]]: _descriptor_ }.
+        1. If _kind_ is `"field"`, set _element_.[[Initializer]] to _initializer_.
+        1. Return _element_.
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id=sec-to-element-finisher-extras aoid=ToElementFinisherExtras>
+      <h1>ToElementFinisherExtras ( _elementDescriptor_ )</h1>
+      <p>With parameter _elementDescriptor_, returns a Record containing three values: { [[Element]]: ElementDescriptor, [[Extras]]: an iterable of other Element Descriptors, [[Finisher]]: a Function or *undefined* }.</p>
+      <emu-alg>
+        1. Let _element_ be ? ToElementDescriptor(_elementDescriptor_).
```

Comment:
Should we also assert that _elementDescriptor_ is an Object, or at least that it is an ECMAScript language value?